src/components/GlobalHeader.js

```diff
@@ -66,15 +68,23 @@ const GlobalHeader = ({ className }) => {
               <ExternalLink
                 href={`${githubBaseUrl}/blob/master/${fileRelativePath}`}
               >
-                <FeatherIcon name="edit" size="1rem" />
+                <FeatherIcon name="edit" size="1rem" className={styles.icon} />
               </ExternalLink>
             </li>
           )}
           <li className={styles.rightSideButton}>
             <ExternalLink href={`${githubBaseUrl}/issues/new/choose`}>
-              <FeatherIcon name="github" size="1rem" />
+              <FeatherIcon name="github" size="1rem" className={styles.icon} />
             </ExternalLink>
           </li>
+          <li className={styles.rightSideButton}>
+            <FeatherIcon
+              name={darkMode.value ? 'sun' : 'moon'}
```

Comment:
Nitpick: I noticed that, when the site first loads, the moon icon is displayed momentarily. I suspect this is due to `darkMode.value` being falsy. We may want to not show the icon at all until we have access to that variable. It seems like the OSS site operates this way.

Reply:
I might log this as a separate issue. I think it has to do with the fact that its a pre-rendered page, which doesn't have access to the local storage/media queries. Even if we hide it, there will still be a flash of content, which would be nice to avoid altogether.

Thoughts? Or should I try and solve this in this PR?

Reply:
Turns out it's more complicated than I thought. I logged this issue in #385. Its annoying, but not a blocker for now, so lets fix at a later time. This will allow us to get this out the door sooner.